core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -27,13 +27,17 @@
 import alluxio.worker.block.BlockWorker;
 import alluxio.worker.block.meta.BlockMeta;
 
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
```

Comment:
unfortunately checkstyle doesn't support this sort of enforcement. In Eclipse you can set "Organize Imports" like [this](http://www.alluxio.org/documentation/resources/eclipse_imports.png) and in Intellij you can set the import code style like [this](http://www.alluxio.org/documentation/resources/intellij_imports.png).